backend/btrixcloud/operator.py

```diff
@@ -133,6 +135,29 @@ def __init__(self):
         with open(self.config_file, encoding="utf-8") as fh_config:
             self.shared_params = yaml.safe_load(fh_config)
 
+        self.compute_crawler_resources()
+
+    def compute_crawler_resources(self):
+        """compute memory / cpu resources for crawlers"""
+        # pylint: disable=invalid-name
+        p = self.shared_params
+        num = max(int(p["crawler_browser_instances"]) - 1, 0)
+        if not p.get("crawler_cpu"):
+            base = parse_quantity(p["crawler_cpu_base"])
+            extra = parse_quantity(p["crawler_extra_cpu_per_browser"])
```

Comment:
Should we do this in a try/except block to catch invalid quantity values?

Reply:
Or maybe just throwing an exception is fine since it'll show up in the logs...

Reply:
Yeah, and it'll throw immediately on container startup so will see it write away in the logs (tested invalid values).